mirror-godot-app/ui/notifications/notify.gd

```diff
@@ -1,14 +1,16 @@
-## Notify: Autoload singleton for notifications, not dependent on GameUI.
+## Notify: Autoload singleton for notifications, not dependent on GameUI.instance.
 extends Node
 
 
 var _notifications_ui: NotificationsUI = null
 
 
 func _ready() -> void:
+	## TODO: Fix this to use a signal to wait for the UI, this code is dumb
+	await GameUI.ui_ready()
 	await get_tree().process_frame
-	if Zone.is_client() and has_node(^"/root/GameUI"):
```

Comment:
This code should really check if notifications should be enabled, since we essentially removed what the original intent was doing.